stdlib/public/SwiftShims/module.modulemap

```diff
@@ -16,7 +16,7 @@ module SwiftShims {
   header "System.h"
   header "ThreadLocalStorage.h"
   header "UnicodeShims.h"
-  header "Visibility.h"
+  textual header "Visibility.h"
```

Comment:
This module map can't be changed; it's the one that ships to customers for use in C mode. We'll have to split out the C++ parts of HeapObject.h, RefCount.h, and Visibility.h.